sql/src/main/java/io/crate/execution/engine/collect/sources/ShardCollectSource.java

```diff
@@ -385,6 +388,11 @@ private CrateCollector createMultiShardScoreDocCollector(RoutedCollectPhase coll
             }
         }
 
+        List<DataType> columnTypes = new ArrayList<>(collectPhase.toCollect().size());
```

Comment:
Can be replaced with ``Symbols.typeView(collectPhase.toCollect()`` and be inlined below (line 406)

Reply:
Ah sweet, I didn't know about this